doc/api/http.md

```diff
@@ -593,8 +593,11 @@ is finished.
 ### request.finished
 <!-- YAML
 added: v0.0.1
+deprecated: REPLACEME
 -->
 
+> Stability: 0 - Deprecated.
```

Comment:
There should be a recommendation on how to avoid hitting deprecated API and to migrate.
Either somewhere in this block, or just by adding a link to the corresponding entry in `deprecations.md`.
@nodejs/documentation -- thoughs?

Reply:
how do I add a link?